openvpn/middlewares/client/bytescount/middleware.go

```diff
@@ -22,14 +22,17 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/mysteriumnetwork/node/client/stats"
```

Comment:
It feels like this dependency should not be here. Some deep internal openvpn package (middleware) knows about one of possible stats consumers. Maybe as an alternative, statistics and state callbacks can be extracted as separate interfaces in top openvpn package and reused by both sides

Reply:
I imagine 2 packages in future:
- go-openvpn - the one which we opensource for anybody to use
- Openvpn Adapter - our specific openvpn integration to Mysterium Node

So, it should belong to Openvpn Adapter. I suggest `openvpn/session/stats`

Reply:
Openvpn adapter should become openvpn service provider plugin :)